Cargo.toml

```diff
@@ -16,7 +16,11 @@ members = [
     "crates/engine/id-derives",
     "crates/engine/id-newtypes",
 ]
-exclude = ["crates/wasi-component-loader/examples", "examples/hooks-template", "examples/query_plan"]
+exclude = [
+    "crates/wasi-component-loader/examples",
```

Comment:
are these excludes still needed?

Reply:
When adding the query_plan one needed it for sure, did you try compiling it?

Reply:
```
❯ cargo build
error: current package believes it's in a workspace when it's not:
current:   /home/brabier/github/grafbase/grafbase/examples/query_plan/subgraph/Cargo.toml
workspace: /home/brabier/github/grafbase/grafbase/Cargo.toml

this may be fixable by adding `examples/query_plan/subgraph` to the `workspace.members` array of the manifest located at: /home/brabier/github/grafbase/grafbase/Cargo.toml
Alternatively, to keep it out of the workspace, add the package to the `workspace.exclude` array, or add an empty `[workspace]` table to the package's manifest.
```

Reply:
so yes they're still necessary

Reply:
Ah true, I only built from the workspace side, didn't try to build the examples. Will fix.